pkg/ddl/backfilling_dist_scheduler.go

```diff
@@ -446,6 +457,7 @@ func splitSubtaskMetaForOneKVMetaGroup(
 			DataFiles:      dataFiles,
 			StatFiles:      statFiles,
 			RangeSplitKeys: rangeSplitKeys,
+			TS:             ts,
```

Comment:
For DXF, if we use the same TS, is there any chance that two keys are conflicts but they cannot be detected by `CollectRemoteDupRows`?

Reply:
It's unlikely because two same keys are adjacent in global sort files, and they will be included in one subtask because start/end key of subtask can not separate them.